### PR TITLE
Various minor fixes and reformatting

### PR DESCRIPTION
**Pull request by phil-s**

### CODE REVIEW

**Comment by phil-s** [comment]
I suggest also doing https://github.com/rougier/mu4e-thread-folding/issues/8 after this.

File: mu4e-thread-folding.el [hunk 1]
```diff
@@ -150,8 +151,8 @@ This uses the mu4e private API and this might break in future releases."
         (let ((overlay-priority     -60) 
               (folded               (if (string= mu4e-thread-folding-default-view 'folded) t nil))
               
-              (child-overlay        nil)
-              (child-prefix-overlay nil)
+              (_child-overlay        nil)
```
Comment (rougier) [inline mu4e-thread-folding.el:154]: Why the `_`?

File: mu4e-thread-folding.el [hunk 2]
```diff
@@ -180,11 +181,11 @@ This uses the mu4e private API and this might break in future releases."
                                           (+ child-prefix-end (line-beginning-position))))
 
                    ;; Overlay for child (whole line)
-                   (child-overlay  (make-overlay
-                                    (+ 0 (line-beginning-position))
-                                    (+ 1 (line-end-position)))))
                
-               ;; We mark the root thread if and only if there'a child               
+               ;; We mark the root thread if and only if there's child
+                   (child-overlay (make-overlay
```
Comment (rougier) [inline mu4e-thread-folding.el:184]: Not sure I understand why you moved this part

File: mu4e-thread-folding.el [hunk 3]
```diff
@@ -56,7 +56,8 @@
 (require 'color)
 
 (defgroup mu4e-thread-folding '()
-  "Group for mu4e thread folding options")
+  "Group for mu4e thread folding options"
+  :group 'mu4e)
```
Comment (rougier) [inline mu4e-thread-folding.el:60]: ok

**Comment by rougier** [comment]
Maybe you can also apply #8, we'll save one commit! (just revert my local changes)

File: mu4e-thread-folding.el (hunk 1 above)
Comment (phil-s) [reply mu4e-thread-folding.el:154]: These are lexical variables which aren't used at all, so the byte-compiler complains.

The `_` marks them as *intentionally* unused, which suppresses the warning.

File: mu4e-thread-folding.el (hunk 2 above)
Comment (phil-s) [reply mu4e-thread-folding.el:184]: Double space removed:

```diff
-child-overlay  (make-overlay
+child-overlay (make-overlay
```

Comment (phil-s) [reply mu4e-thread-folding.el:184]: Wait, something strange has happened there.  I will fix and re-push.

Comment (phil-s) [reply mu4e-thread-folding.el:184]: Sorry about that.  I'd done some initial reformatting on an older revision of the code, and I obviously messed up when rebasing it.

**Comment by phil-s** [comment]
> Maybe you can also apply #8, we'll save one commit! (just revert my local changes)

Done, and pushed.

File: mu4e-thread-folding.el (hunk 1 above)
Comment (phil-s) [reply mu4e-thread-folding.el:154]: You could alternatively not bind these at all; but I figured that maybe you were planning to use them later.

Comment (rougier) [reply mu4e-thread-folding.el:154]: I see. the _child can be removed I think. For the root, I'll need to enforce initial folding (which is buggy right now)

**Pull request merged**